base_kanban_stage_state/tests/test_base_kanban_stage.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Specialty Medical Drugstore
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBaseKanbanStage(TransactionCase):
+    def test_get_states(self):
+        """It should return a list of stages"""
+        test_stage = self.env["base.kanban.stage"].with_context({})
```

Comment:
```suggestion
        test_stage = self.env["base.kanban.stage"]
```
I think this is not needed.